code/game/gamemodes/clock_cult/clock_structures/traps/brass_skewer.dm

```diff
@@ -0,0 +1,88 @@
+//Non-servants standing over this will get spikes through the feet, immobilizing them until they're freed.
+/obj/structure/destructible/clockwork/trap/brass_skewer
+	name = "brass skewer"
+	desc = "A massive brass spike, cleverly concealed in the floor. You think you could prevent setting it off if you moved slowly."
```

Comment:
The second part is more a reference to how pressure sensors work, though they're probably the best way to trigger these.